mods/ra/rules/ai.yaml

```diff
@@ -245,9 +245,9 @@ Player:
 		ProductionTypes: barr,tent,weap,afld,hpad
 		NavalProductionTypes: spen, syrd
 		SiloTypes: silo
-		DefenseTypes: hbox,pbox,gun,ftur,tsla,agun,sam
```

Comment:
the AI should still build them, just perhaps reduce the frequency

Reply:
I can do that  - except for tsla - I found that really impacted the performance of the Ai due to power management issues.

Reply:
if it's an issue then this is not the correct way of solving it. Instead AI's power management needs to be fixed directly

Reply:
it's also a cost thing, for whatever reason I've never seen the AI build a flametower that early only a tsla coil which is a very bad strategy.  Even if I put a massive delay on the tesla coil it still comes first..

Reply:
try puttin a delay on tesla then

Reply:
I guess I was wrong, a massive delay does work.  SAM sites get created first.

Updated PR with that, thanks!